src/components/resources/assort/ResourceRedirect.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from "@radix-ui/themes";
+import React from "react";
+
+interface ResourceRedirectProps {
+  link: string;
+  text: string;
+}
+
+export const ResourceRedirect: React.FC<ResourceRedirectProps> = ({
+  link,
+  text,
+}) => {
+  return (
+    <Link
+      href={`https://www.${link}`}
```

Comment:
**Question:**  Are we sure all those sites use the `www` subdomain?  Seems better to just pass through the full href.